src/core/jslib.c

```diff
@@ -339,11 +339,11 @@ EM_JS_BOOL(bool, JsvFunction_Check, (JsVal obj), {
 });
 
 EM_JS_VAL(JsVal, JsvFunction_CallBound, (JsVal func, JsVal this_, JsVal args), {
-  return nullToUndefined(func.apply(this_, args));
+  return nullToUndefined(Function.prototype.apply.apply(func, [ this_, args ]));
```

Comment:
apply.apply looks very funny 🤔

Reply:
We could pretty reasonably say `Function.prototype.apply.call(func, this_, args)` but it would look less funny.

Reply:
but it'd be nicer on the *JS* *GC* 😇